gui/system/models.py

```diff
@@ -761,9 +761,10 @@ class CertificateBase(Model):
         verbose_name=_("Subject Alternate Names"),
         help_text=_("Multi-domain support. Enter additional space separated domains")
     )
-    cert_serial = models.IntegerField(
+    cert_serial = models.CharField(
```

Comment:
We should convert it back to integer in middlewared extend method. I can commit to this PR if you would like.

Reply:
Also since its now a char, we probably need validation its an integer on the form?

Reply:
Can you elaborate on converting back to int? I don't see anywhere where that would be necessary. As for the form validation, this particular field is never exposed in the UI, it's strictly internal, so it's not necessary to do form validation.

Reply:
I've changed the maximum to 48 in latest commit.

Reply:
In the cert_extend method, it will output it as string, if I am not mistaken.

For the form validation, we do allow serial to be set in Import CA, is that not necessary?

Reply:
The import form gets it from the certificate. There is no place to manually enter it.

Reply:
Whats that field, then? https://i.imgur.com/St5iggR.png

Reply:
That is the identifier, a human name for the certificate to be recognized as and used in the UI for all the things that require certificates. It can be whatever you'd like it be.

Reply:
Thats `Identifier`, not `Serial`.

Reply:
Serial was removed at some point, I'm unsure when. I am running FreeNAS-11.2-MASTER-201805080407 to do this work, are you up to date?

Reply:
I am not, I am unable to update until the TN lab is back up. If the serial field is gone, then my apologies.